### ISSUE TITLE
Tengai on Steam

### ISSUE THREAD

**Issue by stuckinstyx** [farmerbb/RED-Project#196]
I picked up the Psykyo  Shooters on Steam during the Winter Sale to see if I can get some roms out of them.  Tengai has all of the files sitting in a folder called Tengai.  If I zip them up into tengaij,zip it works with the fbneo core in retroarch.

None of the other Psykyo shooters I bought seem to have the rom files exposed like that.  I'll do some digging to see if I can find them.

**Comment by RealRelativeEase** [comment]
This also seems to work with Samurai Aces, according to this: https://milkchoco.info/archives/9263

**Comment by farmerbb** [comment]
Yes, I can confirm that this can be done with Samurai Aces - compress everything inside the SNGKACE folder and name it sngkace.zip.
